app/views/miq_policy/_alert_list.html.haml

```diff
@@ -1,8 +1,10 @@
 #alert_list_div
-  - if x_active_tree == :alert_tree && @view
-    - if @view.table.data.empty?
-      - msg = _("No Alerts are defined %s.") % @search_text.blank? ? "" : _("that match the entered search string")
-      = render :partial => 'layouts/info_msg', :locals => {:message => msg}
+  - if @view.table.data.empty?
+    - if @search_text.empty?
+      - msg = _("No Alerts are defined")
     - else
-      #main_div
-        = render :partial => 'layouts/gtl', :locals  => {:action_url => "alert_get_all", :button_div => 'policy_bar'}
+      - msg = _("No Alerts are defined that match the entered search string '%s'") % @search_text
+    = render 'layouts/info_msg', :message => msg
```

Comment:
@GregP Don't we need the `locals` hash here?
`= render :partial => 'layouts/info_msg', :locals => {:message => msg}`